src/api/application/application.ts

```diff
@@ -340,6 +340,17 @@ export class Application extends EmitterBase<ApplicationEvents> {
         return this.wire.sendAction('set-application-zoom-level', Object.assign({}, this.identity, { level })).then(() => undefined);
     }
 
+    /**
+     * Sets a username to correlate with App Log Management.
+     * @param { string } userName Username to correlate with App's Log.
+     * @return {Promise.<void>}
+     */
+    public async setAppLogUsername(userName: string): Promise<void> {
+        await this.wire.sendAction('set-app-log-username', Object.assign({}, this.identity, {
```

Comment:
:neckbeard: `Object.assign({data: userName}. this.identity)`. No need to clone into an empty object when you're already creating a new one for the payload.